concent_api/core/message_handlers.py

```diff
@@ -391,8 +400,12 @@ def handle_send_force_subtask_results_response(request, client_message):
     if subtask.subtask_results_accepted_id is not None or subtask.subtask_results_rejected_id is not None:
         raise Http400("This subtask has been resolved already.")
 
-    verification_deadline = subtask.ack_report_computed_task.timestamp.timestamp() + settings.SUBTASK_VERIFICATION_TIME
-    acceptance_deadline = verification_deadline + settings.FORCE_ACCEPTANCE_TIME + settings.CONCENT_MESSAGING_TIME
+    acceptance_deadline = (
+        computation_deadline +
+        settings.SUBTASK_VERIFICATION_TIME +
+        settings.FORCE_ACCEPTANCE_TIME +
+        settings.CONCENT_MESSAGING_TIME
+    )
```

Comment:
I know it is not your change but can you please reverse the condition below, to make it like in other places ?

Reply:
You mean switching `if current_time > acceptance_deadline:` to `if acceptance_deadline < current_time:` ?

Reply:
Done